docker/package/Dockerfile-fedora

```diff
@@ -27,6 +27,7 @@ COPY protocols.json /tezos-packaging/protocols.json
 COPY docker/package/*.py /tezos-packaging/docker/package/
 COPY docker/package/defaults /tezos-packaging/docker/package/defaults
 COPY docker/package/scripts /tezos-packaging/docker/package/scripts
+COPY tezos_baking /tezos-packaging/docker/package/tezos_baking
```

Comment:
IMO `tezos_baking` is not the right top-level directory nor name for this new project, especially because the point of this issue is to distance it from the `tezos-baking` package

Reply:
@krendelhoff2 I see that you've changed this, but `code` is also not that great of a name for this, as it's too generic and also because it doesn't help with the separation that we are looking for.

So, how about simply calling the top-level dir `baking` (but keeping `tezos-baking` as the python package name, as you did now)?